File: Nível 1/2375 - Sedex/main.lua
diametroBola = tonumber(io.read())
infos = io.read()
alturaC, larguraC, profundC = infos:match("(%S+)%s+(%S+)%s+(%S+)")
alturaC, larguraC, profundC = tonumber(alturaC), tonumber(larguraC), tonumber(profundC)
cabe = true 

if diametroBola > alturaC or diametroBola > larguraC or diametroBola > profundC then
    cabe = false  
end

if cabe == true then
    print("S")
else
    print("N")
end

File: Nível 1/2375 - Sedex/main.py
diametroBola = int(input("A: "))
alturaC, larguraC, profundC = map(int, input().split())

cabe = True

if(diametroBola > alturaC or diametroBola > larguraC or diametroBola > profundC):
    cabe = False 

if(cabe == True):
    print("S")
else:
    print("N")